jwst/associations/tests/test_main.py

```diff
@@ -17,12 +17,12 @@
 
 @pytest.mark.parametrize(
     'case', [
-        (None, 62),  # Don't re-run, just compare to the generator fixture results
+        (None, 65),  # Don't re-run, just compare to the generator fixture results
         (['-i', 'o001'], 2),
         (['-i', 'o001', 'o002'], 3),
-        (['-i', 'c1001'], 1),
-        (['-i', 'o001', 'c1001'], 3),
-        (['-i', 'c1001', 'c1002'], 2),
+        (['-i', 'c1001'], 0),
```

Comment:
@stscieisenhamer Can't easily tell what exactly these particular tests are checking for, so I'm not sure if the fact that some of them are now returning a zero count is OK. Any thoughts?

Reply:
Bad commentary on the test. I'll check to see what is the intent and suggest changes.